install/kubernetes/helm/istio/values.yaml

```diff
@@ -579,38 +583,12 @@ tracing:
     tag: 1.7
     memory:
       max_traces: 50000
-    ingress:
```

Comment:
this seems fine to me, however, we have PRs that want to improve on this functionality.  I'm not really clear what this does - just provide an old-school Kubernetes ingress?

We have a design doc in the env wg team that is covering multiple profiles - one profile of which is nginx + istio (iiuc).  In this model, ingressgateway wouldn't be present, and the addons wouldn't work correctly.

Is that correct @linsun?

Reply:
correct, we can't assume ingress gateway is always there.

Reply:
@linsun should the old ingress be added back in as it was?

Reply:
@linsun Have added the ingresses back in, but just wondering if the `.Values.jaeger.ingress.enabled` and `.Values.ingress.enabled` should be replaced by the `global.k8sIngress.enabled`? Are they representing the same thing?